projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -113,3 +113,73 @@ export async function jobdDataRequest(): Promise<void> {
     );
   }
 }
+
+/**
+ * Sent a POST request to the server to get more information about the notebook that is failing the job run
+ * Returns the path to the notebook file and the index of the cell that is failing in the notebook
+ */
```

Comment:
May you elaborate the function documentation, regarding failingCellId argument? For example, what happens with different values of failingCellId passed? Like, empty, non-failing-cell, failing-cell?